Podfile

```diff
@@ -8,23 +8,23 @@ target :'AsyncDisplayKitTests' do
   pod 'JGMethodSwizzler', :git => 'https://github.com/JonasGessner/JGMethodSwizzler', :branch => 'master'
 
   # Only for buck build
-  pod 'PINRemoteImage', '3.0.0-beta.7'
+  pod 'PINRemoteImage', '3.0.0-beta.10'
+end
 
-  #TODO CocoaPods plugin instead?
-  post_install do |installer|
-    require 'fileutils'
+#TODO CocoaPods plugin instead?
+post_install do |installer|
+  require 'fileutils'
```

Comment:
Since the post_install hook isn't associated with the target (i.e. you can only have one per file,) moved this to root.